pages/opening-hours.js

```diff
@@ -44,9 +44,12 @@ const OpeningHours = (props) => (
               error: props.t(props.validatorErrors.opening_hours_monday)
             }}
           >
-            <Heading as="h4" mb={1} size="MEDIUM">
+            <Label
```

Comment:
I don't think this needs to be a <Label>.  The govuk <InputField> element already wraps it all in a label for you.